### DISCUSSION TITLE
how to add custom header?

### DISCUSSION THREAD

**Discussion by mrguamos** [moov-io/iso8583#252] [Q&A]
I need to append some value after the length prefix, is this possible? 
In java j8583 library there is something like this
[IsoMessage#setIsoHeader](https://j8583.sourceforge.net/apidocs/com/solab/iso8583/IsoMessage.html#setIsoHeader(java.lang.String))

**Comment by mrguamos** [comment]
nvm, i just prepend it with the raw message

**Reply by alovak** [reply]
@mrguamos please check the link below to see how you can use custom message writer here - so you can write anything you need before writing the message

https://github.com/moov-io/iso8583-connection/pull/44
